experiments/ppo_gridnet_eval.py

```diff
@@ -170,17 +167,17 @@ def parse_args():
             global_step += 1 * args.num_envs
             # ALGO LOGIC: put action logic here
             with torch.no_grad():
-                invalid_action_masks[step] = torch.tensor(np.array(envs.get_action_mask())).to(device)
+                invalid_action_masks[0] = torch.tensor(np.array(envs.get_action_mask())).to(device)
```

Comment:
If we're always indexing at index 0, maybe we don't need this dimension at all? Then this line of code would just look like `invalid_action_masks = ...`, i.e. a direct assignment, not assignment into a slot indexed by 0. But maybe that would mean that we're always allocating new memory, instead of writing to a pre-allocated block? I really don't know enough of the deep technical details of PyTorch yet to be sure here, hopefully you are able to comment @vwxyzjn